static/js/apps/event/app.tsx

```diff
@@ -91,20 +96,41 @@ export function App(props: AppPropsType): JSX.Element {
   tableProperties.sort();
 
   const dateDisplay = getDateDisplay(props.properties);
+  let placeBreadcrumbsJsx: JSX.Element[];
+  if (props.place.dcid) {
+    const allPlaces = [props.place, ...props.parentPlaces];
+    placeBreadcrumbsJsx = allPlaces.map((place, i) => (
+      <>
+        <a className="place-links" href={`/disasters/${place.dcid}`}>
+          {place.name}
+        </a>
+        {i < allPlaces.length - 1 ? ", " : ""}
+      </>
+    ));
+  }
+
+  const provenanceJsx = props.provenance.map((p, i) => (
+    <>
+      <a href={p.url}>{p.sourceName}</a>
+      {i < props.provenance.length - 1 ? ", " : ""}
+    </>
+  ));
 
   return (
     <RawIntlProvider value={intl}>
       <Container>
         <div className="head-section">
           <h1>{props.name}</h1>
-          <h3>type: {typeOf[0].name}</h3>
-          <h3>
-            dcid: <a href={`/browser/${props.dcid}`}>{props.dcid}</a>
-          </h3>
           <h3>
-            source:{" "}
-            <a href={`/browser/${provenance[0].dcid}`}>{provenance[0].name}</a>
+            <span>{_.startCase(typeOf[0].name)}</span>
+            {placeBreadcrumbsJsx && <> in {placeBreadcrumbsJsx}</>}
           </h3>
+          <h4 className="clearfix">
```

Comment:
nit: curious why this classname

Reply:
there's a float on the browser link in the element, so the clearfix "resets" the floats 

https://getbootstrap.com/docs/4.0/utilities/clearfix/